tests/api_test.py

```diff
@@ -86,14 +87,15 @@ class TestSigstoreSigning:
     def test_sign_and_verify(
         self, sigstore_oidc_beacon_token, sample_model_folder, tmp_path
     ):
-        sc = model_signing_api.SigningConfig()
+        sc = sign.SigningConfig()
```

Comment:
Is reducing repetition as much as a thing in Python as it is in Go?

eg. 
`sign.SigningConfig` vs. `sign.Config`
similar with:
`hash.HashingConfig` vs `hash.Config`